v1/websocket.go

```diff
@@ -8,7 +8,7 @@ import (
 	"net/http"
 	"reflect"
 
-	"github.com/bitfinexcom/bitfinex-api-go/utils"
+	"github.com/blockcapital/bitfinex-api-go/utils"
```

Comment:
I don't think some of these lines are supposed to be included.  Your earlier opening of #128 indicates you might be having a bit of trouble with the github PR interface.

How about explaining what changes you are trying to submit and someone might be able to assist.